### PR TITLE
feat: improve folder org and tmux conf 

### PR DESCRIPTION
**Pull request by tagoro9**
**Description**

This PR reorganizes all the folder structure so it is easier to follow and it simplifies the tmux config to not depend on a third party config that is doing so much. 

**Changes**

* feat: change theme to one light
* refactor: better organize files
* feat(aliases): add 1password alias
* fix(brew): add missing dep
* docs(readme): remove nord theme instructions
* fix(script): only change shell when it&#39;s needed
* chore(deps): add urlview
* feat(tmux): simplify config
* fix(install): rem unused alias
* feat(tmux): simplify config again
* feat(tmux): configure dracule powerline

🚀 PR created with [fotingo](https://github.com/tagoro9/fotingo)

### CODE REVIEW

**Review by llosimura — APPROVED** [review]

File: config/tmux/tmux.conf [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+run -b 'tmux bind -T copy-mode-vi H send -X start-of-line 2> /dev/null || true'
+run -b 'tmux bind -t vi-copy L end-of-line 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi L send -X end-of-line 2> /dev/null || true'
+
+# Plugins
+set -g @plugin 'tmux-plugins/tpm'
+set -g @plugin 'tmux-plugins/tmux-sensible'
+set -g @plugin 'tmux-plugins/tmux-urlview'
+set -g @plugin 'tmux-plugins/tmux-yank'
+set -g @plugin 'dracula/tmux'
```
Comment (llosimura) [inline config/tmux/tmux.conf:70]: are you using dracula?

Comment (tagoro9) [reply config/tmux/tmux.conf:70]: Yeah! It seemed to be the best I could find and also copied it from someone haha

File: Brewfile [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
-brew "diff-so-fancy"
-brew "jenv"
-brew "pyenv"
-brew "tmux"
+brew "awscli"
 brew "bat"
-brew "k9s"
-brew "kubectx"
+brew "cli53"
+brew "diff-so-fancy"
```
[the remark this answers is not in the document]
Comment (tagoro9) [reply Brewfile]: 🚀

File: config/tmux/tmux.conf [hunk 3]
```diff
@@ -45,6 +45,9 @@ bind t select-layout tiled
 # enable mouse navigation
 set -g mouse on
 
+# use C-a for nested sessions
+bind-key -n C-a send-prefix
```
Comment (tagoro9) [inline config/tmux/tmux.conf:49]: This breaks vim command to increase a number 😢

**Pull request merged**